### PR TITLE
goreleaser: use it

### PR DESCRIPTION
**Pull request by greut**
[goreleaser](https://goreleaser.com/) is a fantastic tool to package go programs. Let's use it.

### CODE REVIEW

File: cmd/editorconfig/main.go [hunk 1]
```diff
@@ -13,10 +13,8 @@ import (
 	"github.com/editorconfig/editorconfig-core-go/v2"
 )
 
-const (
-	// Version indicates the current version number
-	Version = "2.1.1"
-)
+// Version indicates the current version number
+const Version = "2.1.1"
```
Comment (andreynering) [inline cmd/editorconfig/main.go]: Shouldn't this constant be converted into a variable and be populated automatically by Goreleaser?

Comment (greut) [reply cmd/editorconfig/main.go]: I've tried that... but `core-test` wants a real value. It could probably be faked in the Makefile.

Comment (andreynering) [reply cmd/editorconfig/main.go]: No problem then 👍

Comment (greut) [reply cmd/editorconfig/main.go]: I've put a fake version in the Makefile.

**Comment by greut** [comment]
I've rebased this and feel like it could be used.

**Pull request merged**